ibm/resource_ibm_cm_offering_instance.go

```diff
@@ -7,11 +7,21 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"time"
 
 	"github.com/IBM/platform-services-go-sdk/catalogmanagementv1"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+const (
+	inProgress = "in progress"
+	failed     = "failed"
+	success    = "succeeded"
+
+	waitUntilInterval = 10 * time.Second
```

Comment:
Please make use of schema timeouts instead of hardcoding here..
Please refer: https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/resource_ibm_container_alb_cert.go#L26